lib/srv/forward/sshserver_test.go

```diff
@@ -0,0 +1,109 @@
+/*
+ *
+ * Copyright 2022 Gravitational, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * /
+ */
+
+package forward
+
+import (
+	"crypto/rand"
+	"testing"
+
+	apisshutils "github.com/gravitational/teleport/api/utils/sshutils"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/crypto/ssh"
+)
+
+func Test_signersWithSHA1Fallback(t *testing.T) {
```

Comment:
Is this a new standard for naming tests? I've been doing `TestName`, but see that some have been doing `Test_name`.

Reply:
I used my IDE to generate that test. I'll rename it.